src/core/model_config.proto

```diff
@@ -661,6 +726,40 @@ message ModelDynamicBatching
   //@@     order.
   //@@
   bool preserve_ordering = 3;
+
+  //@@  .. cpp:var:: uint32 priority_levels
+  //@@
+  //@@     The number of priority levels to be enabled for the model,
+  //@@     the priority level starts from 1. Requests with lower priority level
```

Comment:
"... start from 1 and 1 is the highest priority. Requests are handled in priority order with all priority 1 requests processed before priority 2, all priority 2 processed before priority 3, etc. Requests with the same..."